src/pip/_internal/commands/install.py

```diff
@@ -30,9 +34,16 @@
     protect_pip_from_modification_on_windows,
 )
 from pip._internal.utils.temp_dir import TempDirectory
+from pip._internal.utils.typing import MYPY_CHECK_RUNNING
 from pip._internal.utils.virtualenv import virtualenv_no_global
 from pip._internal.wheel import WheelBuilder
 
+if MYPY_CHECK_RUNNING:
+    from optparse import Values
+    from typing import Any, List
+
+    from pip._internal.req.req_install import InstallRequirement
+
 logger = logging.getLogger(__name__)
```

Comment:
I would put two empty lines after imports and before body contents.

Reply:
Done in the other file as well.